go/teams/teams.go

```diff
@@ -412,17 +402,25 @@ func (t *Team) implicitTeamDisplayName(ctx context.Context, skipConflicts bool)
 			isFullyResolved = false
 		case keybase1.TeamInviteCategory_KEYBASE:
 			// Check to make sure we don't already have the user in the name
-			iname := string(invite.Name)
```

Comment:
a bit tricky... technically invite.Name for keybase-type is uv percentform, but AnnotateInvites( overwrote it with the username. I shuffled the types mainly because this was hard to follow

Reply:
Don't understand this comment. The invite is not AnnotatedInvite here....

Reply:
Not anymore. It used to be